pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -493,6 +460,50 @@ func (r *ReconcileIBMBlockCSI) reconcileServiceAccount(instance *ibmblockcsi.IBM
 			if err != nil {
 				return err
 			}
+
+			controllerStatefulset, err := r.getControllerStatefulSet(instance)
+			if err != nil {
+				return err
+			}
+
+			nodeDaemonSet, err := r.getNodeDaemonSet(instance)
+			if err != nil {
+				return err
+			}
+
+			if config.GetNameForResource(config.CSIControllerServiceAccount, instance.Name) == sa.Name {
```

Comment:
please give this return value a name, before comparing it.
e.g. `controllerServiceAccountName` (same for node)

Reply:
done